benchmarks/benchmarks.rb

```diff
@@ -132,9 +124,7 @@ def positional_splat_keyword_and_double_splat_args(a, *args, b:, **kwargs)
 #
 # NOTE: The proportion of falsey results is 1/N_UNIQUE_ARGUMENTS (because for
 # the methods with arguments we are truthy for all but the first unique argument
-# set, and for zero-arity methods we manually execute `no_args` N_TRUTHY_RESULTS
-# times per each execution of `no_args_falsey`). This number was selected as the
-# lowest number such that this logic:
```

Comment:
@JacobEvelyn why remove the falsey tests from the benchmarks -- just not clear if this is on purpose?